packages/components/src/slot-fill/bubbles-virtually/slot.tsx

```diff
@@ -13,12 +17,25 @@ import { useMergeRefs } from '@wordpress/compose';
 /**
  * Internal dependencies
  */
+import { View } from '../../view';
 import SlotFillContext from './slot-fill-context';
+import type { WordPressComponentProps } from '../../ui/context';
+import type { SlotComponentProps } from '../types';
 
 function Slot(
-	{ name, fillProps = {}, as: Component = 'div', ...props },
-	forwardedRef
+	props: WordPressComponentProps<
+		Omit< SlotComponentProps, 'bubblesVirtually' >,
+		'div'
+	>,
+	forwardedRef: ForwardedRef< any >
 ) {
+	const {
+		name,
+		fillProps = {},
+		children,
```

Comment:
When `ReactNode` is passed to `children` in `Slot`, it is rendered, so remove `children` from `restProps`.